tests/ext/test_jobqueue.py

```diff
@@ -492,12 +492,10 @@ async def test_equality(self, job_queue):
         job_2 = job_queue.run_repeating(self.job_run_once, 0.2)
         job_3 = Job(self.job_run_once, 0.2)
         job_3._job = job.job
-        assert job == job
```

Comment:
why? I'm rather sure that we do want to assert that … same below

Reply:
```PLR0124 Name compared with itself, consider replacing `job == job` ```

should I just ignore it then?

Reply:
yes, please